Gemfile

```diff
@@ -4,33 +4,33 @@ eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 # VMDB specific gems
 #
 
+gem "activerecord-deprecated_finders", "~>1.0.4",  :require => "active_record/deprecated_finders"
 gem "rails",                           RAILS_VERSION
-gem "activerecord-deprecated_finders", "~>1.0.4",     :require => "active_record/deprecated_finders"
 
 # Client-side dependencies
-gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
-gem 'angular-ui-bootstrap-rails', '~> 0.13.0'
-gem 'momentjs-rails', '~> 2.10.3'
-gem 'jquery-rails', "~>4.0.4"
-gem 'jquery-hotkeys-rails'
-gem 'codemirror-rails', "=4.2"
-gem 'lodash-rails', '~> 3.10.0'
+gem "angular-ui-bootstrap-rails",     "~>0.13.0"
+gem "angularjs-rails",                "~>1.4.3"
+gem "codemirror-rails",               "=4.2"
+gem "jquery-hotkeys-rails"
+gem "jquery-rails",                   "~>4.0.4"
+gem "jquery-rjs",                     "=0.1.1",                       :git => "https://github.com/amatsuda/jquery-rjs.git"
+gem "lodash-rails",                   "~>3.10.0"
+gem "momentjs-rails",                 "~>2.10.3"
 
 # On MS Windows run "bundle config --local build.libv8 --with-system-v8" first
```

Comment:
This line should go, if we're cleaning up, since we're no longer depending on v8 - more info in #3773.